keel-api/src/test/kotlin/com/netflix/spinnaker/keel/rest/ArtifactControllerTests.kt

```diff
@@ -59,24 +60,6 @@ internal class ArtifactControllerTests {
       .andExpect(status().isCreated)
   }
 
-  @Test
```

Comment:
I removed this behavior so I removed this test. My thought was that people are going to register their artifacts by submitting an image yaml file. I think we need a better way to indicate that two apps care about the same image.